wordcloud/wordcloud.py

```diff
@@ -679,3 +684,45 @@ def __array__(self):
 
     def to_html(self):
         raise NotImplementedError("FIXME!!!")
+
+    def _get_bolean_mask(self, mask):
+        """Cast to two dimensional boolean mask."""
+        if mask.dtype.kind == 'f':
+            warnings.warn("mask image should be unsigned byte between 0"
+                            " and 255. Got a float array")
+        if mask.ndim == 2:
+            boolean_mask = mask == 255
+        elif mask.ndim == 3:
+            # if all channels are white, mask out
+            boolean_mask = np.all(mask[:, :, :3] == 255, axis=-1)
+        else:
+            raise ValueError("Got mask of invalid shape: %s"
+                                % str(mask.shape))
+        return boolean_mask
+
+    def draw_contour(self, img, mask, contour_width=1, contour_color='black'):
```

Comment:
Shouldn't this be private? Or why do you want it to be public? If it's public, it needs to document its parameters (even if it's private, it probably should).

Reply:
Made it private!